extensions/wikia/CloseMyAccount/CloseMyAccountSpecialController.class.php

```diff
@@ -174,6 +174,10 @@ public function reactivate() {
 			return;
 		}
 
+		// call to User::newFromConfirmationCode() is basically a token check
+		// tell CSRFDetector that we're fine here (PLATFORM-2206)
+		\Wikia\Security\CSRFDetector::onUserMatchEditToken();
```

Comment:
I'd consider `onUserMatchEditToken()` (or hook handlers in general) as private. Having that assumption in mind we're basically creating a new public entry point and I'd still strive to find a more intuitive name for it (eg. `markEditTokenChecked` or something similar)